docker-compose.yml

```diff
@@ -9,3 +9,53 @@ services:
       - ./:/spftest/
     container_name: spftest_dev
     working_dir: /spftest
+  mock-server:
```

Comment:
we had this in the docker-services to reduce the containers run by default.  is there a reason we moved it?

Reply:
@imranq2 sorry not sure what you mean. This project (SAM.Testing) has one docker-compose file (`docker-compose.yml`) to test the new testing framework we need new docker services e.g. mock-server. maybe you're talking about helix.pipeline project?